build.gradle

```diff
@@ -0,0 +1,60 @@
+subprojects { project ->
+
+  repositories {
+    mavenCentral()
+  }
+
+  apply plugin: 'checkstyle'
+
+  task checkstyle(type: Checkstyle) {
+    configFile rootProject.file('checkstyle.xml')
+    source 'src/main/java'
+    ignoreFailures false
+    showViolations true
+    include '**/*.java'
+
+    classpath = files()
+  }
+
+  afterEvaluate {
+    if (project.tasks.findByName('check')) {
+      check.dependsOn('checkstyle')
+    }
+  }
+
+  buildscript {
+    repositories {
+      mavenCentral()
+    }
+    dependencies {
+      classpath 'com.android.tools.build:gradle:1.3.1'
+      classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
+    }
+  }
+}
+
+ext {
+  minSdkVersion = 15
+  targetSdkVersion = 23
+  compileSdkVersion = 23
+  buildToolsVersion = '23.0.1'
+  sourceCompatibilityVersion = JavaVersion.VERSION_1_7
+  targetCompatibilityVersion = JavaVersion.VERSION_1_7
+}
+
+ext.deps = [
+    // Android
+    android: 'com.google.android:android:4.1.1.4',
```

Comment:
@JakeWharton I can move this to a testCompile dependency for the compiler module if it's preferred.